docs/arcadia-users-group/20240730-arcadia-pycolor/lesson.md

```diff
@@ -0,0 +1,225 @@
+# Using `arcadia-pycolor` to make figures in Python
+
+`arcadia-pycolor` is a Python package that provides a set of color palettes and convenience functions to style plots so that they are compatible with Arcadia's style guide. This notebook provides a quick introduction to `arcadia_pycolor` how to use it to style `matplotlib` and `seaborn` plots so that they comply with the Arcadia style guide, largely following the [Quickstart Guide](https://github.com/Arcadia-Science/arcadia-pycolor/blob/main/docs/quickstart.md).
+
+We'll cover the following topics:
+
+1. Installing `arcadia-pycolor`
+2. Configuring `matplotlib` defaults
+3. Accessing and using colors, palettes, and gradients
+4. Using `arcadia-pycolor` to style plots
+
+
+## 1. Install the `arcadia-pycolor` package
+
+We've designed `arcadia-pycolor` to be distributed via PyPI. In a virtual environment of your choice, run the following command in your terminal:
+
+```bash
+pip install arcadia-pycolor
+```
+
+### Requirements
+`arcadia-pycolor` requires Python >=3.9 and has the following dependencies (which are detected and installed automatically when you install `arcadia-pycolor` using `pip`):
+- `matplotlib` (>=3.7,!=3.8.0)
+- `colorspacious` (>=1.1.2)
+
+Additionally, `arcadia-pycolor` checks whether you have the **"Suisse Int'l"** family of fonts installed on your system. If these fonts aren't installed, `arcadia-pycolor` will default to **"Arial"**. You can download the "Suisse Int'l" font family from [this Google Drive link](https://drive.google.com/drive/folders/1xMLKDiaFVVg81xdS03uqsggjzkYyMrqJ?usp=drive_link) (must be accessed using an `@arcadiascience.com` email).
```

Comment:
Just want to flag if it's fine we have the link here (even though it's only accessible with arcadiascience.com email domains) or we just have the link somewhere else.

Reply:
Just to be extra safe, I removed the link and instead directed people to the "Brand Assets" page in Notion; I'll share the relevant links live during the lesson.